developer-docs-site/docs/guides/use-aptos-explorer.md

```diff
@@ -0,0 +1,265 @@
+
+The Aptos Explorer provides a one-step search engine across the blockchain to discover details about wallets, transactions, network analytics, user accounts, smart contracts, and more. The Aptos Explorer also offers dedicated pages for key elements of the blockchain and acts as the source of truth for all things Aptos. See the [Aptos Glossary](../reference/glossary.md) for definitions of many of the terms found here.
+
+## Users
+
+The Aptos Explorer gives you a near real-time view into the status of the network and the activity related to the core on-chain entities. It serves these audiences and purposes by letting:
+
+* App developers understand the behavior of the smart contracts and sender-receiver transaction flows.
+* General users view and analyze Aptos blockchain activity on key entities - transactions, blocks, accounts, and resources.
+* Node operators check the health of the network and maximize the value of operating the node.
+* Token holders find the best node operator to delegate the tokens and earn a staking reward.
+
+## Common tasks
+
+Follow the instructions here to conduct typical work in the Aptos Explorer.
+
+### Select network
+
+The Aptos Explorer renders data from all Aptos networks: Mainnet, Testnet, Devnet, and your local host if configured. See [Aptos Blockchain Deployments](../nodes/aptos-deployments.md) for a detailed view of their purposes and differences.
+
+To select a network in the [Aptos Explorer](https://explorer.aptoslabs.com/), load the explorer and use the *Select Network* drop-down menu at the top right to select your desired network.
+
+<center>
+<ThemedImage
+alt="Select Network in Aptos Explorer"
+sources={{
+    light: useBaseUrl('/img/docs/0-explorer-select-network.png'),
+    dark: useBaseUrl('/img/docs/0-explorer-select-network-dark.png'),
+  }}
+/>
+</center>
+
+### Find a transaction
+
+One of the most common tasks is to track a transaction in Aptos Explorer. You may search by the account address, transaction version and hash, or block height and version.
+
+To find a transaction:
+
+1. Enter the value in the *Search transactions* field near the top of any page.
+1. Do not press return.
+1. Click the transaction result that appears immediately below the search field, highlighted in green within the following screenshot:
+
+<center>
+<ThemedImage
+alt="Search Aptos Explorer for a transaction"
+sources={{
+    light: useBaseUrl('/img/docs/1-explorer-search-txn.png'),
+    dark: useBaseUrl('/img/docs/1-explorer-search-txn-dark.png'),
+  }}
+/>
+</center>
+
+The resulting [Transaction details](#transaction-details) page appears.
+
+### Find account address
+
+The simplest way to find your addess is to use the [Aptos Petra Wallet](https://petra.app/docs/use).
+
+Then simply append it to the following URL to load its details in th Aptos Explorer:
+https://explorer.aptoslabs.com/account/
```

Comment:
I think this is confusing because this results in a link in the doc and if a user clicks on it, it will take them to an empty state (since there is no account address appended as the default) 

I think it might be worth it to showcase a screenshot example here (the flow is to paste the address into the 'search explorer' field on the homepage of the explorer)

Reply:
I code formatted the account/ URL to prevent it from auto-linking. Please take a look.